### PR TITLE
Breaking PHP Open API changes are now in a dedicated chapter; added a…

### PR DESCRIPTION
**Pull request by zeitgen**
… new chapter on PhpStorm 2020.3

### CODE REVIEW

File: reference_guide/api_changes/api_changes_list_2020.md [hunk 1]
```diff
@@ -210,7 +215,7 @@ Java EE plugins split
 ### Changes in PhpStorm and PHP Plugin 2020.2
 
 Added Union Types Support
-: Please see [PhpStorm Breaking Changes](/products/phpstorm/php_open_api_breaking_changes.md).
+: See [Breaking Changes in PhpStorm 2020.2](/products/phpstorm/php_open_api_breaking_changes_2002.md).
```
Comment (YannCebron) [inline reference_guide/api_changes/api_changes_list_2020.md:218]: broken link /products/phpstorm/php_open_api_breaking_changes_2003.md

File: _SUMMARY.md [hunk 2]
```diff
@@ -252,6 +252,8 @@
 * [PhpStorm](products/phpstorm/phpstorm.md)
     * [Working with the PHP Open API](products/phpstorm/php_open_api.md)
         * [Breaking Changes](products/phpstorm/php_open_api_breaking_changes.md)
+            * [2020.2](products/phpstorm/php_open_api_breaking_changes_2002.md)
```
Comment (YannCebron) [inline _SUMMARY.md:255]: can we name the page "2020_2" or "202" etc.?

**Review by YannCebron — CHANGES_REQUESTED** [review]

File: _SUMMARY.md (hunk 2 above)
Comment (zeitgen) [reply _SUMMARY.md:255]: I think we can, why not.

File: reference_guide/api_changes/api_changes_list_2020.md (hunk 1 above)
Comment (zeitgen) [reply reference_guide/api_changes/api_changes_list_2020.md:218]: Missed that, thanks!

File: _SUMMARY.md (hunk 2 above)
Comment (YannCebron) [reply _SUMMARY.md:255]: and put latest on top, thanks

**Pull request merged**